code/src/components/AddTodo.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { tasks } from 'reducers/tasks';
+import styled from 'styled-components';
```

Comment:
I like the way you worked with styled components inside each one of them and also used GlobalStyle for this project.